src/extension/background-script/actions/ln/makeInvoice.ts

```diff
@@ -20,7 +21,9 @@ const makeInvoice = async (message, sender) => {
       });
       return response;
     } catch (e) {
-      return { error: e.message };
+      if (e instanceof Error) {
```

Comment:
what happens if e is not an Error? will this exception be swallowed?

I think it would be great to have a helper function we can use to get a message from an exception

Reply:
Didn't you mention this recently somewhere else? Wanna create an issue for this?

Reply:
It's because of TypeScript; it should always be an error (according to the makeInvoice function)

But yeah, we use this repeatedly and it's best to have a helper function which somehow extracts the message from exception var. Not sure how we do that, though.